test/core/iomgr/tcp_server_posix_test.c

```diff
@@ -454,7 +454,7 @@ int main(int argc, char **argv) {
   const grpc_channel_args channel_args = {1, chan_args};
   struct ifaddrs *ifa = NULL;
   struct ifaddrs *ifa_it;
-  test_addrs dst_addrs;
+  static test_addrs dst_addrs;
```

Comment:
Please either add a comment calling this out, or (preferably) heap allocate this memory.